templates/server/src/main/resources/archetype-resources/pom.xml

```diff
@@ -287,7 +287,7 @@
         <plugin>
           <groupId>org.springframework.boot</groupId>
           <artifactId>spring-boot-maven-plugin</artifactId>
-          <version>2.0.4.RELEASE</version>
+          <version>2.1.6.RELEASE</version>
```

Comment:
fine but same as above: should be `${spring.boot.version}`

Reply:
Done